core/templates/components/molecules/ctas/ctas.json

```diff
@@ -0,0 +1,8 @@
+{
+  "grid": "grid--4-of-12",
```

Comment:
Do you expect others to be able to change this?

Reply:
This is the part I'm unsure about how to handle - elements that are not 100% width/things that would probably occupy a certain number of grid columns depending on how the site builder wants it. Let me know how I should structure this so it make sense. Thanks!